project/scripts/sbt

```diff
@@ -1,26 +1,20 @@
 #!/usr/bin/env bash
 
-CI_TEST=$1
+# Usage:
+# ./sbt <cmd>
 
-if [ -n "$2" ]; then
-  if [ $2 = true ]; then
-    SHOULD_RUN=true
-  else
-    SHOULD_RUN=false
-  fi
-else
-  SHOULD_RUN=true
+CMD="$1"
+
+if [ -z "$CMD" ]; then
+  echo "Error: missing sbt command"
+  exit 1
 fi
 
-if [ $SHOULD_RUN = true ]; then
-  # get the ivy2 cache:
-  ln -s /var/cache/drone/ivy2 "$HOME/.ivy2" || true
```

Comment:
@felixmulder What was `|| true` for?

Reply:
I think this was just there to allow `ln` to fail, having a cache is not the most important thing to this script. It could definitely be removed, but we then loose the ability to test without `/var/cache/drone/ivy2` being present - not a big deal I guess.

Reply:
I had the `ln` command run as an initial step before the tests but it was not persisted between the steps and the CI started re-downloading artifacts.  So I added it in all the CI scripts

Reply:
We used to have a `common` script that was sourced from the other scripts - if there's too much duplication, you could consider doing something similar eventually.

Reply:
Note that `ln` also fails if "$HOME/.ivy2" already exists, so I think you might get spurious error messages in the log upon running the script the second time on the same container.

Reply:
Ah, true! That's exactly, I remember now!

Reply:
Does `ln` survive multiple steps? Because I tried to do it once before all the steps, but then the CI would redownload artifacts: http://dotty-ci.epfl.ch/lampepfl/dotty/102

Reply:
I will experiment

Reply:
It doesn't seem so  🤔

Reply:
It does however survive within a single `command` section, IIRC